spec/support/credentials.rb

```diff
@@ -3,22 +3,34 @@
 Dotenv.load
 
 def creds
-  OpenStruct.new(
-    {
-      consumer_key: ENV['QBO_API_CONSUMER_KEY'],
-      consumer_secret: ENV['QBO_API_CONSUMER_SECRET'],
-      token: ENV['QBO_API_ACCESS_TOKEN'],
-      token_secret: ENV['QBO_API_ACCESS_TOKEN_SECRET'],
-      realm_id: ENV['QBO_API_COMPANY_ID']
-    }
-  )
+  if ENV['QBO_API_OAUTH2_ACCESS_TOKEN']
```

Comment:
Method is too long. Rename the existing `creds` method to `oauth1_creds` and then in the `creds` method have a simple conditional that calls either `oauth1_creds` or `oauth2_creds`.